common/federation/src/main/java/app/coronawarn/server/common/federation/client/DefaultHostnameVerifierProvider.java

```diff
@@ -0,0 +1,18 @@
+
+
+package app.coronawarn.server.common.federation.client;
+
+import javax.net.ssl.HostnameVerifier;
+import org.apache.http.conn.ssl.DefaultHostnameVerifier;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("!disable-ssl-client-federation-verify-hostname")
```

Comment:
reuse profile name with "efgs" instead of "federation", might be confusing otherwise